src/status_im/ui/screens/views.cljs

```diff
@@ -97,26 +97,29 @@
     :component-will-unmount utils.universal-links/finalize
     :reagent-render
     (fn []
-      [react/safe-area-provider
-       ^{:key (str @colors/theme @reloader/cnt)}
-       [react/view {:flex             1
-                    :background-color colors/black-persist}
-        [navigation/navigation-container
-         (merge {:ref               (fn [r]
-                                      (navigation/set-navigator-ref r))
-                 :onStateChange     on-state-change
-                 :enableURLHandling false}
-                (when debug?
-                  {:enableURLHandling true
-                   :initialState      @state}))
-         [main-app-navigator]]
-        [wallet/prepare-transaction]
-        [wallet/request-transaction]
-        [wallet/select-account]
-        [signing/signing]
-        [bottom-sheet]
-        [popover/popover]
-        (when debug?
-          [reloader/reload-view @reloader/cnt])
-        (when config/keycard-test-menu-enabled?
-          [hardwallet.test-menu/test-menu])]])}))
+      (let [disbale-touch-events?
+            @(re-frame/subscribe [:disable-touch-events?])]
+        [react/safe-area-provider
+         ^{:key (str @colors/theme @reloader/cnt)}
+         [react/view {:flex             1
+                      :background-color colors/black-persist
+                      :pointer-events disbale-touch-events?}
```

Comment:
shouldn't it be something like this `:pointer-events (if disbale-touch-events? 'none' 'auto')` ?

Reply:
youre right, though for some reason it works with just true/false

Reply:
weird

Reply:
interesting, but it might stop working in next RN versions

Reply:
sure, fixed